src/styles.css

```diff
@@ -4,7 +4,7 @@ body {
 }
 
 .header {
-  height: 50px;
+  height: 49px;
```

Comment:
This is not correct! 

You should leave it as 50px.

(Not really.. this is just a test message)